test/opt/copy_prop_array_test.cpp

```diff
@@ -1884,6 +1884,81 @@ OpFunctionEnd
   SetTargetEnv(SPV_ENV_UNIVERSAL_1_4);
   SinglePassRunAndMatch<CopyPropagateArrays>(before, false);
 }
+
+TEST_F(CopyPropArrayPassTest, StructIndexCannotBecomeDynamic) {
```

Comment:
Add a comment that explains what this test is testing, possibly referencing the instructions in the spir-v.  It will help if somebody has to update the test later.

Reply:
Added